test/api/workflows_format_2/converter.py

```diff
@@ -1,14 +1,13 @@
-import os
-import sys
+"""Functionality for converting a Format 2 workflow into a standard Galaxy workflow."""
+from __future__ import print_function
 
-import yaml
+from collections import OrderedDict
```

Comment:
I usually put `from foo import bar` imports after `import foo` imports (in each of the 3 recommended subsections), but I can't find an official source for this practice. Anyone has opinions on this?